### PR TITLE
open-cluster-management/backlog#2427 -- quite a bit of rewrite after …

### PR DESCRIPTION
**Pull request by swopebe**
…doc test, WIP

### CODE REVIEW

File: manage_applications/deleting_channel.adoc [hunk 1]
```diff
@@ -3,24 +3,27 @@
...
- .. Within the search box, filter by `kind:channel` to view all channels.
- .. Within the list of all channels, select the _Options_ menu for the channel that you want to delete.
-Click *Delete channel*.
- .. When the list of all channels is refreshed, the channel is no longer displayed.
-* To use the Kubernetes CLI tool,
- .. Run the following command to delete the channel from a target namespace.
-Replace `name` and `namespace` with the name of your channel and your target namespace:
+To use the console:
+
+. Click on _Subscriptions_.
```
Comment (swopebe) [inline manage_applications/deleting_channel.adoc]: I didn't see Delete channel

File: manage_applications/managing_apps_console.adoc [hunk 2]
```diff
@@ -131,17 +121,15 @@ Any changes that you save are applied to the resource immediately.
 
 For more information about using search, see link:../console[Search in the console].
 
-[#topology]
-== Topology
+[#resource-topology]
+== Resource topology
+
+The topology provides you with a visualization of your applications, clusters, relationships, and policies. The application topology includes a visualization of the components that comprise each application.
```
Comment (birsanv) [inline manage_applications/managing_apps_console.adoc]: This was changed in 2.0, we don't the the app topology, we show app cards with application status

Change : 
The application topology includes a visualization of the components that comprise each application. 
with, sample:
The application topology includes a visualization of all applications cards showing application status

File: manage_applications/managing_apps_console.adoc [hunk 3]
```diff
@@ -131,17 +121,15 @@ Any changes that you save are applied to the resource immediately.
...
+The topology provides you with a visualization of your applications, clusters, relationships, and policies. The application topology includes a visualization of the components that comprise each application. 
 
-The Topology page provides you with a visualization of your applications, clusters, relationships, and policies.
-The application topology includes a visualization of the components that comprise each application.
 The topology view for each application includes any deployables, services, deployments, charts, and pods for that application.
 
-You can select any component from the topology view to view more details.
-You can also hover your cursor over a resource to view the component kind, name, and namespace and links to view the search results for the resource or namespace.
-When you are viewing the details for a pod, you can select to view the logs for that pod.
+* You can select any component from the topology view to view more details.
```
Comment (birsanv) [inline manage_applications/managing_apps_console.adoc]: The Topology tab for app doesn't show app topology, just app cards

File: manage_applications/managing_placement_rules.adoc [hunk 4]
```diff
@@ -14,15 +14,15 @@ Samples for all resources, including placement rules, are located in the xref:..
 The custom resource definition (CRD) and controller for placement rules replaces the placement policies that were used for applications in previous versions of Red Hat Advanced Cluster Management for Kubernetes.
 Placement policies are still used for governance and risk policies.
 
-Placement rules can be defined for subscriptions and for deployables.
-Define the placement rule at the subscription level for multi-cluster deployments.
+Placement rules can be defined for placement rules and for deployables.
+Define the placement rule at the placement rule level for multi-cluster deployments.
```
Comment (birsanv) [inline manage_applications/managing_placement_rules.adoc]: Define the placement rule at the **subscription** level

File: manage_applications/managing_placement_rules.adoc [hunk 5]
```diff
@@ -14,15 +14,15 @@ Samples for all resources, including placement rules, are located in the xref:..
...
 Define the placement rule for a specific deployable for single-cluster deployments or to override placement settings.
 
 [#create-a-placement-rule]
 == Create a placement rule
 
-Placement rules can be defined for subscriptions and for deployables.
-Define the placement rule at the subscription level for multi-cluster deployments.
+Placement rules can be defined for placement rules and for deployables.
+Define the placement rule at the placement rule level for multi-cluster deployments.
```
Comment (birsanv) [inline manage_applications/managing_placement_rules.adoc]: subscription level

File: manage_applications/managing_placement_rules.adoc [hunk 6]
```diff
@@ -31,12 +31,12 @@ You can run `oc get pods -n open-cluster-management-agent-addon` to check for po
 . Compose the definition YAML content for your placement rule.
 Samples for all resources, including placement rules, are located in the xref:../manage_applications/app_sample.adoc#application-samples[Application resource samples]documentation.
 . Create the placement rule within Red Hat Advanced Cluster Management for Kubernetes.
-You can define a placement rule as a separate resource or define the rule within the definition for a deployable or subscription.
+You can define a placement rule as a separate resource or define the rule within the definition for a deployable or placement rule.
```
Comment (birsanv) [inline manage_applications/managing_placement_rules.adoc]: I thought we are not discussing deployables anymore .. the Placement Rule is always defined as a separate resource for app scenarios; the subscription references a placement rule

File: manage_applications/managing_placement_rules.adoc [hunk 7]
```diff
@@ -31,12 +31,12 @@ You can run `oc get pods -n open-cluster-management-agent-addon` to check for po
...
 As a best practice, define placement rules as a separate resource when the rule might need to be referenced by multiple resources.
 +
-For more information about creating the definition for a deployable or subscriptions, see:
+For more information about creating the definition for a deployable or placement rules, see:
```
Comment (birsanv) [inline manage_applications/managing_placement_rules.adoc]: For more information about creating the definition for a  placement rule, see

File: manage_applications/managing_placement_rules.adoc [hunk 8]
```diff
@@ -53,7 +53,7 @@ The _Create a placement rule_ editor is displayed.
   ... When you are finished adding or editing the YAML, click *Save* to create the placement rule.
   ... From the list of resource summary cards, click the _PLACEMENT RULES_ card.
 The _Search_ dashboard is displayed and lists all placement rules that are used by the applications that are selected on the _Applications_ dashboard.
-Your new placement rule does not display in this list until the rule is referenced by a subscription that is used by an application.
+Your new placement rule does not display in this list until the rule is referenced by a placement rule that is used by an application.
```
Comment (birsanv) [inline manage_applications/managing_placement_rules.adoc]: until the rule is referenced by a **subscription**

File: manage_applications/managing_placement_rules.adoc [hunk 9]
```diff
@@ -76,10 +76,10 @@ Ensure that your new placement rule is listed in the resulting output.
 [#assign-a-placement-rule]
 == Assign a placement rule
 
-You can assign a placement rule to a deployable or subscription.
-To assign a placement rule, you need to update the spec for the deployable or subscription to reference the placement rule.
+You can assign a placement rule to a deployable or placement rule.
```
Comment (birsanv) [inline manage_applications/managing_placement_rules.adoc]: take this out

File: manage_applications/managing_placement_rules.adoc [hunk 10]
```diff
@@ -76,10 +76,10 @@ Ensure that your new placement rule is listed in the resulting output.
...
+To assign a placement rule, you need to update the spec for the deployable or placement rule to reference the placement rule.
 
-Include the following `placement` fields in your deployable or subscription spec with the values to reference the placement rule:
+Include the following `placement` fields in your deployable or placement rule spec with the values to reference the placement rule:
```
Comment (birsanv) [inline manage_applications/managing_placement_rules.adoc]: in your subscription spec

File: manage_applications/managing_placement_rules.adoc [hunk 11]
```diff
@@ -90,22 +90,22 @@ placement:
...
 . From the Navigation menu, click *Manage applications*.
 The *Overview* tab for all applications opens.
 . Click the *Resources* tab.
 . Scroll to the *Resource pipeline* section.
-Within the table that lists your applications, expand the row for the application that includes the subscription that is assigned the placement rule.
-. From the expanded view for the application, you can see the available subscriptions for each channel.
-The details for each subscription include any assigned placement rule.
-If needed, you can select to view or edit the YAML for the placement rule, subscription, and channel from this resource pipeline table.
+Within the table that lists your applications, expand the row for the application that includes the placement rule that is assigned the placement rule.
+. From the expanded view for the application, you can see the available placement rules for each channel.
```
Comment (birsanv) [inline manage_applications/managing_placement_rules.adoc]: you can see the available subscriptions

File: manage_applications/managing_placement_rules.adoc [hunk 12]
```diff
@@ -90,22 +90,22 @@ placement:
...
+The details for each placement rule include any assigned placement rule.
```
Comment (birsanv) [inline manage_applications/managing_placement_rules.adoc]: The details for each subscription..

File: manage_applications/managing_placement_rules.adoc [hunk 13]
```diff
@@ -90,22 +90,22 @@ placement:
...
+If needed, you can select to view or edit the YAML for the placement rule, placement rule, and channel from this resource pipeline table.
```
Comment (birsanv) [inline manage_applications/managing_placement_rules.adoc]: the subscription, placement rule, and channel

File: manage_applications/managing_placement_rules.adoc [hunk 14]
```diff
@@ -155,51 +155,51 @@ Alternatively, you can select to edit the YAML from the Applications dashboard r
 The *Overview* tab for all applications opens.
  .. Click the *Resources* tab.
  .. Scroll to the *Resource pipeline* section.
-Within the table that lists your applications, expand the row for the application that includes the subscription that is assigned the placement rule.
- .. From the expanded view for the application, you can see the available subscriptions for each channel.
-The details for each subscription include any assigned placement rule.
+Within the table that lists your applications, expand the row for the application that includes the placement rule that is assigned the placement rule.
```
Comment (birsanv) [inline manage_applications/managing_placement_rules.adoc]: Too many occurrences where subscription was replaced by PR ..

expand the row for the application that includes the placement rule that is assigned the placement rule.

should be 

expand the row for the application that includes the subscription that is assigned the placement rule.

**Review by birsanv — CHANGES_REQUESTED** [review]

File: manage_applications/managing_placement_rules.adoc (hunk 6 above)
Comment (swopebe) [reply manage_applications/managing_placement_rules.adoc]: @birsanv I keep getting back/forth information ... this file is about managing placement rules. If that is not what this file is for anymore, I need development to read and go over the doc again in another issue to discuss use of deployable in this file. (Last I heard, keep it in but take out the tasks for it, which was carried over from IBM)

File: manage_applications/deleting_channel.adoc (hunk 1 above)
Comment (swopebe) [reply manage_applications/deleting_channel.adoc]: addressed

File: manage_applications/managing_apps_console.adoc (hunk 2 above)
Comment (swopebe) [reply manage_applications/managing_apps_console.adoc]: Changed

File: manage_applications/managing_apps_console.adoc (hunk 3 above)
Comment (swopebe) [reply manage_applications/managing_apps_console.adoc]: removed

File: manage_applications/managing_placement_rules.adoc (hunk 10 above)
Comment (swopebe) [reply manage_applications/managing_placement_rules.adoc]: done

File: manage_applications/managing_placement_rules.adoc (hunk 11 above)
Comment (swopebe) [reply manage_applications/managing_placement_rules.adoc]: done, line 105 I pinged about.

File: manage_applications/managing_placement_rules.adoc (hunk 14 above)
Comment (swopebe) [reply manage_applications/managing_placement_rules.adoc]: Changed. But as I look at all these tasks, I think every task above "Delete placement rule" was not changed for the 2.0 UI. This has to be done in a refresh and we have to stay on top of it as the UI changes for 2.1. For this issue, I edited the Delete tasks only.

File: manage_applications/managing_placement_rules.adoc (hunk 9 above)
Comment (swopebe) [reply manage_applications/managing_placement_rules.adoc]: I am not sure what you mean, the entire Assign a placement rule?

File: manage_applications/managing_placement_rules.adoc (hunk 4 above)
Comment (swopebe) [reply manage_applications/managing_placement_rules.adoc]: no idea what happened here lol, changed.

**Review by birsanv — APPROVED** [review]

**Pull request merged**